### ISSUE TITLE
Unable to run the FasterRCNN with VGG16.

### ISSUE THREAD

**Issue by ignvinay** [giddyyupp/caffe-faster-rcnn-cpp-windows#1]
Hi giddyyupp,
     Well i built caffe model without GPU.
I have turned off Python build and also made CPU only build.
But BUILD_PYTHON_LAYER=1 is ON.
But while running i get following error:

_Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Python (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, FrcnnAnchorTarget, FrcnnProposal, FrcnnProposalTarget, FrcnnRoiData, FrcnnVidData, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, RNN, ROIMaskPooling, ROIPooling, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, SmoothL1Loss, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, VideoData, WindowData)_

Any ideas on how i can resolve this ?
